src/db/sysdb_ops.c

```diff
@@ -4909,7 +4909,6 @@ errno_t sysdb_search_object_by_id(TALLOC_CTX *mem_ctx,
     if (filter == NULL) {
         return ENOMEM;
     }
-
```

Comment:
This line removal does not belong to this patch, can you remove this hunk from the patch?

Reply:
Sorry dint know how these creapt in. On my local checkout on branch I cannot see these.

Reply:
Still present in the current series.